privacy-toolset.php

```diff
@@ -9,20 +9,59 @@
  * @package automattic-privacy-toolset
  */
 
-// TODO setup plugin class
-// TODO standardize function names
-// TODO add noscript GTM code in body
-// TODO better version than current time
+namespace Automattic\PrivacyToolset;
 
-function create_privacy_toolset_block() {
-  register_block_type(__DIR__ . '/block/build');
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
 }
-add_action('init', 'create_privacy_toolset_block');
 
-function privacy_toolset_enqueue_styles() {
-  wp_enqueue_style(
-    'privacy-toolset',
-    __DIR__ . '/block/build/privacy-consent-banner.css?ver=' . time()
-  );
+class PrivacyToolset {
```

Comment:
It's good practice not to have definitions of classes and side effect code in the same file. That means the class file can be included/required/autoloaded without risk of some side effect happening.

If you keep the class in a file on it's own e.g. `src/class-privacy-toolkit.php`, then you also don't need the ABSPATH protection, since if anyone does navigate straight the file, then nothing bad happens. That defensive code can stay in this top-level file which requires the class file and instantiates it.